collective/jsonify/wrapper.py

```diff
@@ -310,8 +310,8 @@ def get_archetypes_fields(self):
                 value = self._get_at_field_value(field)
                 value2 = value
 
-                if not isinstance(value, str):
-                    if isinstance(value.data, str):
+                if value and not isinstance(value, str):
+                    if value and isinstance(value.data, str):
```

Comment:
I think this should be `if value.data and isinstance(value.data, str):`

Reply:
I got a error on value.data because value was none, while exporting a site that was using CMFBibliography.

Reply:
Yes, in the second `if` the `value` is already guaranteed to be true.
Maybe check if `value` has an attribute `data`.

Reply:
like
```
if value and not isinstance(value, str):
    if isinstance(getattr(value, 'data', None), str):
```